README.md

```diff
@@ -32,6 +32,18 @@ The easiest way to start working on the project is to use GNU Make:
 $ make bootstrap
 ```
 
+> Please note that with apple silicon (Mx chips), you may encounter difficulties launching docker compose, notably because some images are not compatible. In this case, you can specify the platform in the docker compose file, as in the example below:
```

Comment:
Can you keep the line length around 80 chars, please ?
It's easier to read, and keeps consistency around the project.